uber/templates/registration/checkin.html

```diff
@@ -0,0 +1,117 @@
+<script type="text/javascript">
+    $(function() {
+        $(".dialog-form").dialog({ autoOpen: false });
+
+        $(".attendee-checkin").button().on("click", function () {
+        attendee_id = $(this).attr('data-attendeeid');
+
```

Comment:
Is this supposed to be a global variable?  Just wondering why there isn't a ``var``.